collect_app/src/test/java/org/odk/collect/android/support/FakeScheduler.java

```diff
@@ -13,6 +13,7 @@
     private Runnable foregroundTask;
     private Runnable backgroundTask;
     private Boolean cancelled = false;
+    private Boolean isProgressUpdating = false;
```

Comment:
I like what's happening here but I think the variable/method should be named something more related to the scheduler than the test here. Maybe `isRepeatRunning`?